options/read.go

```diff
@@ -0,0 +1,108 @@
+package options
```

Comment:
All operations are coming with their own set of options. Now I stumbled upon this and I think it might be a better idea. What do you think? https://infevo.net/fluent-configure-in-golang/#solution

Reply:
yeah, that may work

Reply:
I went using @roblaszczak suggestion to create those options.